assets/js/directives/buy-quick-start.directive.js

```diff
@@ -34,7 +34,11 @@ function buyQuickStart ($rootScope, currency, buySell, Alerts, $interval, $timeo
     scope.inMobileBuy = $rootScope.inMobileBuy;
 
     scope.updateLastInput = (type) => scope.lastInput = type;
-    scope.isPendingTradeState = (state) => scope.pendingTrade && scope.pendingTrade.state === state;
+    scope.isPendingTradeState = (state) => scope.pendingTrade && scope.pendingTrade.state === state && scope.pendingTrade.medium !== 'blockchain';
+
+    if (scope.tradingDisabledReason === 'awaiting_first_trade_completion' && scope.pendingTrade.medium === 'blockchain') {
+      scope.isPendingSellTrade = true;
```

Comment:
This is duplicated logic from `sell-quick-start`. Not critical now, but really should refactor these into a single component next week.